core/generics/shrinkingmap/shrinkingmap.go

```diff
@@ -68,12 +68,12 @@ func New[K comparable, V any](opts ...Option) *ShrinkingMap[K, V] {
 	return shrinkingMap
 }
 
-// Set adds a key-value pair to the map. It returns true if the key exists.
-func (s *ShrinkingMap[K, V]) Set(key K, value V) (updated bool) {
```

Comment:
This changed the logic completely :scream: 
We use the shrining map also in other projects....

Reply:
OK, I checked all projects I know. We don't use the return value by now.